src/rawdog/llm_client.py

```diff
@@ -79,23 +69,28 @@ def get_response(
         self,
         messages: list[dict[str, str]],
     ) -> str:
+        base_url = self.config.get("llm_base_url")
```

Comment:
I think `llm_base_url` should be allowed to be none. If it is then Lite llm will read it from the environment or guess it from model name if it's not set there. The user probably expects `export OPENAI_BASE_URL=localhost:8000` to work and the readme say it will.